Makefile

```diff
@@ -195,5 +194,9 @@ dispatch:
             -H "Content-Type: application/json" https://api.github.com/repos/rancher/rke2-upgrade/dispatches \
             --data '{"event_type": "create_tag", "client_payload": {"tag":"'"${DRONE_TAG}"'"}}'
 
+package-airgap:
+	mkdir -p dist/artifacts
```

Comment:
If this directory is used by any other target, it can be made into it's own target.

```
dist/artifacts:
    mkdir -p $@
```